OpenLIFUData/OpenLIFUData.py

```diff
@@ -1906,6 +1906,8 @@ def load_session(self, subject_id, session_id) -> None:
         self.update_photocollections_affiliated_with_loaded_session()
 
         # Check if there are any photoscans with approved transducer tracking results
+        # TODO: This assumes a single approved photoscan per session. This check needs to be udpated to allow for 
+        # approved_photoscan_id containing multiple photoscan IDs. 
         approved_photoscan_id = self.getParameterNode().loaded_session.get_transducer_tracking_approvals()
```

Comment:
Maybe there was previously a bug here? `get_transducer_tracking_approvals` is used without passing `approved_photoscans_only=True`, and so what it's doing here is to to get _all_ photoscan IDs for which there is an approved tt result, rather than getting _only_ the ones where the photoscan itself is also approved.

(Although I guess it shouldn't ever happen that a tt result is approved while the photoscan isn't, but still.)